flask_rebar/rebar.py

```diff
@@ -33,29 +34,12 @@
 from flask_rebar.request_utils import get_query_string_params_or_400
 from flask_rebar.swagger_generation import SwaggerV2Generator
 from flask_rebar.swagger_ui import create_swagger_ui_blueprint
-
```

Comment:
migrated to a subclass so we could rename `marshal_schema` to `response_body_schema` and add a to-be-deprecated `marshal_schema` property for backward compatibility.  Moved elsewhere in the file to sit with other class defs